Modules/termios.c

```diff
@@ -1315,6 +1313,9 @@ static struct constant {
 #ifdef TIOCTTYGSTRUCT
     {"TIOCTTYGSTRUCT", TIOCTTYGSTRUCT},
 #endif
+#ifdef _POSIX_VDISABLE
+    {"_POSIX_VDISABLE", _POSIX_VDISABLE},
+#endif
 
```

Comment:
```python
#!/usr/bin/python3

# Debian GNU/Linux 12 (bookworm)

import subprocess
import termios

def stty_intr_value():
    out = subprocess.run(['stty', '-a'], stdout=subprocess.PIPE, text=True).stdout
    s = out.replace(";", " ")
    sub = "intr = "
    i = s.find(sub) + len(sub)
    return s[i:].split()[0]

t = termios.tcgetattr(0)
t[6][termios.VINTR] = bytes([termios._POSIX_VDISABLE])
termios.tcsetattr(0, termios.TCSANOW, t)

assert stty_intr_value() == "<undef>"
print("PASSED")
```

Hit `Control-C` after running running above script to see `SIGINT` not being sent.

Please run `stty intr ^c` to restore `Control-C` functionality.